docs/zkapps/o1js/recursion.mdx

```diff
@@ -32,17 +32,9 @@ Kimchi, the proof system that backs o1js, supports arbitrary infinite recursive
 
 More generally, you can use recursion to verify any zero-knowledge program as part of your zkApp.
 
```

Comment:
:::note
As of o1js v0.14.0, zkProgram has moved out of the Experimental namespace and is now available as a top-level import directly. 
Experimental.ZkProgram has been deprecated. 

If you are experiencing issues with zkProgram as documented below, please update o1js to the latest version
:::